.travis.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     - WEB3_PUBLIC_WS_PROVIDER=wss://mainnet.infura.io/ws
     script: "$TRAVIS_BUILD_DIR/app/travis_scripts/visual-tests.sh"
   - stage: Platform Build Staging
-    if: NOT type IN (pull_request)
```

Comment:
Can you explain this change please?

Reply:
This was just so I could test in on the PR level